crates/red_knot_python_semantic/resources/mdtest/annotations/callable.md

```diff
@@ -225,14 +225,16 @@ Using `Concatenate` as the first argument to `Callable`:
 from typing_extensions import Callable, Concatenate
 
 def _(c: Callable[Concatenate[int, str, ...], int]):
-    reveal_type(c)  # revealed: (*args: @Todo(todo signature *args), **kwargs: @Todo(todo signature **kwargs)) -> int
+    # TODO: Should reveal the correct signature
+    reveal_type(c)  # revealed: (...) -> int
```

Comment:
If we think that the signature should still reveal `@Todo` somehow, I can try to retain that property, by making `is_gradual` an enum or introducing another flag.

Reply:
I'd prefer to either keep `...` or have `@Todo(...)` but not retain the current display as that's clearly incorrect.

Reply:
Okay, keeping `...` for now. Thank you.